qiita_pet/handlers/util.py

```diff
@@ -50,7 +50,7 @@ def download_link_or_path(is_local_request, filepath, fp_id, label):
         resp = "<b>%s:</b> %s" % (label, filepath)
     else:
         resp = ('<a class="btn btn-default glyphicon glyphicon-download-alt" '
-                'href="/download/%s" style="word-spacing: -10px;"> %s</a>'
+                'href="%s" style="word-spacing: -10px;"> %s</a>'
```

Comment:
I'm confused about this change. Why is this needed? Isn't this doing the download? In the live system I think this is the mechanism used to offload the download to nginx.